lib/audit-markdown.js

```diff
@@ -21,3 +21,21 @@ renderer.link = function (href, title, text) {
   const html = originalLinkRenderer.call(renderer, href, title, text)
   return isLocal ? html : html.replace(/^<a /, `<a target="_blank" rel="noreferrer noopener nofollow" `)
 }
+
+
+export function sortByAndTransformDate(source) {
```

Comment:
I tried to put this in its own file (since it's unrelated to markdown), but couldn't figure out what I needed to do to make it accessible from within `client/library/library/index.html`